### DISCUSSION TITLE
Upgrade packages to ensure vulnerabilities are covered

### DISCUSSION THREAD

**Discussion by zoid-w** [apache/airflow#31497] [General]
Airflow 2.6.1-python3.10 can be flagged by image scanners due to the following vulnerabilties : 

- CVE-2022-1941
- CVE-2023-28858
- CVE-2023-28859
- CVE-2023-30861
- CVE-2023-32681

Upgrading the libraries causing these vulnerabilities might be a good way forward. Currently I have not encountered any issues when using the upgraded library versions, but probably this needs to pass the test suite.

**Comment by potiuk** [comment]
Great. Looking forward to your PR's fixing those.
